orbit/pkg/scripts/scripts.go

```diff
@@ -65,6 +66,7 @@ func (r *Runner) Run(execIDs []string) error {
 			break
 		}
 
+		log.Debug().Msgf("running script %v", execID)
```

Comment:
I opted to keep those debug logs as they can prove useful to understand what's going on on the host. Let me know if you think it's too much noise and I should remove them.